### DISCUSSION TITLE
Self-nomination for contributor role

### DISCUSSION THREAD

**Discussion by joshuatcasey** [paketo-buildpacks/feedback#113] [Dependencies Team]
I would like to self-nominate as a dependencies contributor as part of the upcoming `libdependencies` and workflow work.

@paketo-buildpacks/dependencies-maintainers can I get a 👍 / 👎 ?

**Comment by brayanhenao** [comment]
@sophiewigmore Waiting for a second approval :D

**Comment by ryanmoran** [comment]
Congrats @joshuatcasey on your Dependencies Contributorship!! 🎉
